### ISSUE TITLE
Forbid too deep related imports

### ISSUE THREAD

**Issue by orsinium** [wemake-services/wemake-python-styleguide#990]
# Rule request

## Thesis

Good:

```python
from my_project import something
from . import something
from .. import something
```

So-so:

```python
from ... import something
```

Bad:

```python
from .... import something
```

## Reasoning

It's really difficult to infer to which directory `....` refers to. One dot is the current dir, 2 dots is one dir upper in the tree, it's easy to catch. 3 dots are difficult but arguably ok, I'm not sure about this case. However, 4 dots is a quite hard riddle.

## Solutions

- Refactor your app to avoid too aggressive nesting. "Flat is better than nested".
- Or use absolute imports if you have no choice.

**Comment by sobolevn** [comment]
We already restrict all relative imports: https://github.com/wemake-services/wemake-python-styleguide/blob/master/wemake_python_styleguide/violations/consistency.py#L150

I guess that this is even a better rule 🙂 
I prefer to always write the full path of imports.
